csdc-6/adcs-simulation/cpp/inc/ConfigurationSingleton.hpp

```diff
@@ -95,7 +91,8 @@ struct AccelerometerConfig : public SensorConfig {
 struct ReactionWheelConfig : public ActuatorConfig {
     ReactionWheelConfig(const YAML::Node &node);
 
-    Eigen::Matrix3f momentOfInertia;
+    //Eigen::Matrix3f momentOfInertia;

```

Comment:
Can you delete this comment?

Reply:
While you're at it should we maybe rename this variable? since is not a matrix anymore can just call it `inertia` or something